### PR TITLE
Add support for llama3.2

### PR DESCRIPTION
**Pull request by Char15Xu**
#### What this PR does / why we need it:
In this PR, we add latest llama3.2 models for both mlx and llama.cpp framework

#### Which issue(s) this PR addresses (if any):

Addresses # None

#### Special notes for your reviewer:
Will do a benchmark on llama3.2 soon

#### Does this PR introduce a user-facing change?

```release-note
Users are now able to use 'llama3.2' as a new model input. 
```

### CODE REVIEW

**Review by zenodflow — CHANGES_REQUESTED** [review]
@Char15Xu Nice - is there benchmark results to add to the Llama.md?

**Comment by Char15Xu** [comment]
full benchmark result for llama3.2 3b and partial result for llama3.2 1b have been added to the llama.md.

File: benchmark/Llama.md [hunk 1]
```diff
@@ -51,3 +51,22 @@
...
+|    dblp-acm    |   62.0   | 62.0  | 62.0 |    -     |       2.6        |      479.03      |
+|  dblp-scholar  |   66.67   |  17.6  | 27.85 |    -     |       2.8        |      446.18       |
+| fodors-zagats  |   48.28   | 63.64  | 54.9 |    -     |       2.5        |      74.57       |
+| itunes-amazon  |   29.55   | 50.0  | 37.14 |    -     |       1.9       |      56.19       |
+| walmart-amazon |   32.31   | 43.52  | 37.09 |    -     |       2.8        |      424.64       |
+
+### Meta-Llama-3.2-1B-Instruct-8bit
+
+|   Benchmark    | Precision | Recall |  F1   | Cost ($) | Throughput (pps) | Total Latency (s) |
+|:--------------:|:---------:|:------:|:-----:|:--------:|:----------------:|:-----------------:|
```
Comment (zenodflow) [inline benchmark/Llama.md:71]: Good results. The 1B model has longer latency, this is a bit surprising.

**Review by zenodflow — APPROVED** [review]
@Char15Xu , could you add a random guess results for each dataset? I think there is a guess option of the Libem toolchain, in a future PR? 

Because some of the Llama results are quite low, we want to know how much they actually improve a random guesser.

**Pull request merged**